hw/ip/otbn/util/shared/operand.py

```diff
@@ -246,18 +246,21 @@ class ImmOperandType(OperandType):
     '''A class representing an immediate operand type'''
     def __init__(self,
                  width: Optional[int],
+                 fixed_offset: int,
```

Comment:
Can we rename this `enc_offset` to make it more obvious that it's an offset applied to the encoded value? I think that would make it easier to talk about the difference it and any PC-relative offset.